source/common/http/http1/codec_impl.cc

```diff
@@ -391,7 +391,7 @@ void ConnectionImpl::completeLastHeader() {
                  current_header_field_.getStringView(), current_header_value_.getStringView());
 
   if (!current_header_field_.empty()) {
-    toLowerTable().toLowerCase(current_header_field_.buffer(), current_header_field_.size());
+    current_header_field_.inlineTransform([](char c) { return static_cast<uint8_t>(tolower(c)); });
```

Comment:
Does this change have a perf implication? Should we do the absl thing here now?

Reply:
Changed to abseil's tolower version (ascii_tolower) now.
Compared performance of absl::to_lower to toLowerCase (not going to commit the test, just adding the results, to save these datapoints). absl is a bit faster, probably due to optimizations, so we should use it.
```
ToLowerTableEnvoyLowerCase/0          2.11 ns         2.11 ns    280473610
ToLowerTableEnvoyLowerCase/1          4.17 ns         4.17 ns    122310238
ToLowerTableEnvoyLowerCase/10         8.98 ns         8.98 ns     78490730
ToLowerTableEnvoyLowerCase/50         26.3 ns         26.3 ns     26710790
ToLowerTableEnvoyLowerCase/200        81.3 ns         81.3 ns      8631624
ToLowerTableAbseilLowerCase/0         1.91 ns         1.91 ns    365400798
ToLowerTableAbseilLowerCase/1         3.68 ns         3.68 ns    190285175
ToLowerTableAbseilLowerCase/10        8.94 ns         8.94 ns     77216159
ToLowerTableAbseilLowerCase/50        24.1 ns         24.1 ns     28907364
ToLowerTableAbseilLowerCase/200       72.6 ns         72.6 ns      9664473
```